srcpkgs/yabar/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'yabar'
+pkgname=yabar
+version=0.4.0
+revision=3
```

Comment:
if the package is just about to be added to the repository, the revision should be 1. Otherwise the template looks good to me.